src/onegov/election_day/screen_widgets/generic.py

```diff
@@ -176,3 +176,29 @@ def get_variables(self, layout):
             'embed': False,
             'model': self.model or layout.model
         }
+
+
+@ElectionDayApp.screen_widget(tag='counted-entities-no', category='generic')
+class CountedEntitiesNoWidget(ModelBoundWidget):
+    tag = 'counted-entities-no'
+    template = """
+        <xsl:template match="counted-entities-no">
+            <span class="{@class}">
+                ${layout.format_number(model.progress[0])}
+            </span>
+        </xsl:template>
+    """
+    usage = '<counted-entities-no class=""/>'
+
+
+@ElectionDayApp.screen_widget(tag='total-entities', category='generic')
+class TotalEntitiesWidget(ModelBoundWidget):
```

Comment:
Please use the same naming for this as the above one (TotalEntitiesNumberWidget)